<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>650</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Encryption/Decryption App</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  rgb(32, 55, 100);

color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="KeyGenerationGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>931</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(85, 85, 255);
border-color: rgb(255, 170, 0);</string>
    </property>
    <property name="title">
     <string>Public/Private keys generation</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="GenerateKeys">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(174, 170, 170);</string>
     </property>
     <property name="text">
      <string>Click here to generate keys</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="MainTitleLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>ASYMMETRIC ENCRYPTION &amp; DECRYPTION APP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="InstructionsGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>921</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(85, 85, 255);
border-color: rgb(255, 170, 0);</string>
    </property>
    <property name="title">
     <string>Instructions :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>40</y>
       <width>889</width>
       <height>70</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1. First generate your keys </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2. Keep your private key secure since the strengh and security of asymmetric encryption rely on keeping the private key well protected.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3. Make your public key availabe ( through your website for instance ) for encryption purposes.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>887</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4. NB : Encrption is done with public key while decryption is done with private key. ( This concept makes the encryption asymmetric.  )</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="BrowseGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>921</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(85, 85, 255);
border-color: rgb(255, 170, 0);</string>
    </property>
    <property name="title">
     <string>Keys and files  selection</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="KeyPath">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>681</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(174, 170, 170);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BrowsePriKey">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(174, 170, 170);</string>
     </property>
     <property name="text">
      <string>Browse key</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="BrowseFile">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(174, 170, 170);</string>
     </property>
     <property name="text">
      <string>Browse file</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="FilePath">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>681</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(174, 170, 170);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="EnDeGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>931</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(85, 85, 255);
border-color: rgb(255, 170, 0);</string>
    </property>
    <property name="title">
     <string>Encryption / Decryption</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="EncryptBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(174, 170, 170);</string>
     </property>
     <property name="text">
      <string>Encrypt</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="DecryptBtn">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(174, 170, 170);</string>
     </property>
     <property name="text">
      <string>Decrypt</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="MainTitleLabel_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>560</y>
      <width>711</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Designed by ELI KOFFIVI AMEGAN ( www.elkoam.site  /  +233548287700)     </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="actionClose_2">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
